src/components/calcite-shell/calcite-shell.tsx

```diff
@@ -53,37 +53,27 @@ export class CalciteShell {
     return hasHeader ? <slot name={SLOTS.header} /> : null;
   }
 
-  renderContent(): VNode {
-    return !!this.contentBehind ? this.renderContentBehind() : this.renderContentInline();
-  }
-
-  renderContentBehind(): VNode {
-    return (
-      <div
-        key="content-behind"
-        class={{
-          [CSS.content]: true,
-          [CSS.contentBehind]: !!this.contentBehind
-        }}
-      >
-        <slot />
-      </div>
-    );
-  }
-
-  renderContentInline(): VNode {
-    return (
-      <div key="content-inline" class={CSS.content}>
-        <slot />
-        {this.renderCenterRow()}
-      </div>
-    );
-  }
-
-  renderCenterRow(): VNode {
-    const hasCenterRow = !!getSlotted(this.el, SLOTS.centerRow);
-
-    return hasCenterRow ? <slot name={SLOTS.centerRow} /> : null;
+  renderContent(): VNode[] {
+    const content = !!this.contentBehind
+      ? [
+          <div
```

Comment:
Do we not need the key on here anymore?

Reply:
It doesn't seem like it was making a difference.